src/subscriptions/SubscriptionManager.cpp

```diff
@@ -329,7 +329,7 @@ SubscriptionManager::unsubProposedTransactions(std::shared_ptr<WsBase> session)
     txProposedSubscribers_.unsubscribe(session);
 }
 void
```

Comment:
Could use a spacer line in between the functions to follow the same style with above